lib/linter/linter.js

```diff
@@ -1607,6 +1593,42 @@ class Linter {
         }
 
         const sourceCode = slots.lastSourceCode;
+
+        /*
+         * Make adjustments based on the language options. For JavaScript,
+         * this is primarily about adding variables into the global scope
+         * to account for ecmaVersion and configured globals.
+         */
+        sourceCode.applyLanguageOptions(languageOptions);
+
+        let inlineConfig = {};
+        const inlineConfigProblems = [];
```

Comment:
`inlineConfigProblems` seems essentially unused, we're only adding elements but then doing nothing with it.

Reply:
Oops, thanks.